modules/custom/cu_profile_module_manager/cu_profile_module_manager.module

```diff
@@ -196,33 +197,34 @@ function cu_profile_module_manager_addon_page() {
     $packages_array = array();
   }
 
-  foreach ($packages as $key => $package) {
-    $tag = _prepared_atlas_package($key, 'tag');
-    $name = _prepared_atlas_package($key, 'name');
-    $label = _prepared_atlas_package($key, 'label');
+  foreach ($packages['_items'] as $package) {
+    $tag = (isset($package['meta']['tag'][0]) ? $package['meta']['tag'][0] : '');
```

Comment:
Need to handle cases where there are several tags and the one you are interested in is not the first tag in the array